src/item.rs

```diff
@@ -125,6 +130,29 @@ mod test {
         assert_eq!(cfg, CONFIG.load(&store).unwrap());
     }
 
+    #[test]
+    fn existance() {
```

Comment:
I suggest naming the tests by function to clarify what's the test and what is boilerplate

```suggestion
    fn exists_works() {
```